client/public/index.html

```diff
@@ -2,6 +2,9 @@
 <html>
   <head>
     <meta charset="UTF-8" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400&family=Lilita+One&family=Montserrat:wght@200;300;400;500;900&family=Ubuntu:wght@300;400;500&display=swap" rel="stylesheet">
```

Comment:
instead of linking fonts, consider importing them in the SCSS
```
@import url('https://fonts.googleapis.com/css?family=Shadows+Into+Light|Quicksand&display=swap');
```